src/docker.rs

```diff
@@ -233,6 +270,28 @@ pub fn image(config: &Config, target: &Target) -> Result<String> {
     Ok(format!("{CROSS_IMAGE}/{target}:{version}"))
 }
 
+#[cfg(target_os = "windows")]
+fn wslpath(path: &Path, verbose: bool) -> Result<PathBuf> {
+    let wslpath = which::which("wsl.exe")
+        .map_err(|_| eyre::eyre!("could not find wsl.exe"))
+        .warning("usage of `env.volumes` requires WSL on Windows")
+        .suggestion("is WSL installed on the host?")?;
+
+    Command::new(wslpath)
+        .arg("-e")
+        .arg("wslpath")
+        .arg("-a")
+        .arg(path)
+        .run_and_get_stdout(verbose)
+        .wrap_err_with(|| {
+            format!(
+                "could not get linux compatible path for `{}`",
+                path.display()
+            )
+        })
+        .map(|s| s.trim().into())
+}
```

Comment:
it would be possible to fallback to a manual implementation of "normalizing". the logic is quite simple but I don't feel like it's necessary to do.

Reply:
I take that back, I tried to do it correctly but handling this is quite annoying :)

Reply:
maybe

```rust
fn wslpath2(path: impl AsRef<Path>) -> Result<String> {
    fn get_path_prefix_drive(path: &Path) -> Option<&str> {
        match path.components().next().unwrap() {
            std::path::Component::Prefix(prefix_component) => match prefix_component.kind() {
                std::path::Prefix::VerbatimDisk(_) => Some(
                    prefix_component
                        .as_os_str()
                        .to_str()
                        .expect("windows drive letters should be ascii A-Z")
                        .strip_prefix(r"\\?\")?
                        .strip_suffix(":")?,
                ),
                std::path::Prefix::Disk(_) => Some(
                    prefix_component
                        .as_os_str()
                        .to_str()
                        .expect("windows drive letters should be ascii A-Z")
                        .strip_suffix(":")?,
                ),
                _ => None,
            },
            _ => None,
        }
    }
    let path = path.as_ref();
    let components = path.components().skip(2);
    let mut path_c = String::from("/mnt/");

    path_c
        .push_str(&get_path_prefix_drive(path).ok_or_else(|| eyre::eyre!("no drive letter found"))?.to_lowercase());
    let mut sep = true;
    for comp in components {
        if sep {
            path_c.push('/');
        } else {
            sep = true;
        }
        match comp {
            std::path::Component::Normal(p) => path_c.push_str(
                p.to_str()
                    .ok_or_else(|| eyre::eyre!("path needs to be utf8"))?,
            ),
            std::path::Component::ParentDir => path_c.push_str(".."),
            std::path::Component::CurDir => path_c.push('.'),
            _ => sep = false,
        }
    }
    Ok(path_c)
}
```